org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/telemetry/DocumentTelemetryInfo.java

```diff
@@ -67,10 +70,25 @@
 						break;
 					}
 				}
+				if (identifier != null) {
```

Comment:
We could probably filter out identifiers that are local. I wonder if we could just exclude anything with a file URI.

Reply:
Done.